.claude/commands/arena.md

```
@@ -0,0 +1,191 @@
+
+Once selected, extract the original camps from the doc and go to [Dispatch Beat Reporters](#dispatch-beat-reporters) with the same framing.
+
+---
+
+## Dispatch Beat Reporters
+
+Launch one `newsroom:beat-reporter` agent per camp, **all in parallel** using the Agent tool with `run_in_background: true`.
+
+### Reporter Brief Template
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)